config/DEMO.ts

```diff
@@ -8,7 +8,4 @@ export default {
         "ELASTICSEARCH": "info",
     },
     "USE_JS_ORM_ENTITIES": true,
-    "WEB3_OPTIONS": {
```

Comment:
Should we keep a generic "ETHEREUM_SERVICE_OPTIONS" setting or something?

Reply:
The `WEB3_OPTIONS` was a hotfix solution that ultimately did not prove successful.  The problem it was attempting to solve was one of the final straws for the migration to ethers.js.  

Right now, there are no additional configurable settings we need to be concerned with.  If that need arises, we can add an `ETHEREUM_SERVICE_OPTIONS` and refactor some of the GETH_* settings at that point as well.  For now, I think that's a little out of scope.